create/cloudvm.go

```diff
@@ -0,0 +1,137 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	runtimev1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	infrastructure "github.com/ninech/apis/infrastructure/v1alpha1"
+	meta "github.com/ninech/apis/meta/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+type cloudVMCmd struct {
+	Name                string            `arg:"" default:"" help:"Name of the CloudVM instance. A random name is generated if omitted."`
+	Location            string            `default:"nine-es34" help:"Location where the CloudVM instance is created."`
+	MachineType         string            `default:"nine-standard-1" help:"MachineType defines the sizing for a particular cloud vm."`
+	Hostname            string            `default:"" help:"Hostname allows to set the hostname explicitly. If unset, the name of the resource will be used as the hostname. This does not affect the DNS name."`
+	PowerState          string            `default:"on" help:"PowerState specifies the power state of the cloud VM. A value of On turns the VM on, shutdown sends an ACPI signal to the VM to perform a clean shutdown and off forces the power off immediately."`
+	OS                  string            `default:"ubuntu22.04" help:"OS which should be used to boot the VM."`
+	BootDisk            map[string]string `default:"{name:\"root\",size:\"20Gi\"}" help:"BootDisk that will be used to boot the VM from. Needs to be in the following format: {name:\"<name>\",size:\"<size>Gi\"}"`
```

Comment:
Although it does not directly reflect the API, I think it would be a bit more user-friendly if we don't allow setting the name of the boot disk and just hardcode it to `root`. Then we can simply have a `BootDiskSize` flag. The name is totally irrelevant anyways.

Reply:
added